pkg/vault/contracts/Vault.sol

```diff
@@ -1179,17 +1176,16 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
                 amountInUnderlying = vaultUnderlyingDelta - bufferUnderlyingSurplus;
                 // Since bufferUnderlyingSurplus was wrapped, the final amountOut needs to discount the wrapped amount
                 // that will stay in the buffer
-                amountOutWrapped = vaultWrappedDelta - bufferWrappedSurplus;
+                amountOutWrapped = vaultWrappedDelta - wrappedToken.convertToShares(bufferUnderlyingSurplus);
```

Comment:
Maybe make `bufferWrappedSurplus` local
```suggestion
                uint256 bufferWrappedSurplus = wrappedToken.convertToShares(bufferUnderlyingSurplus)
                amountOutWrapped = vaultWrappedDelta - bufferWrappedSurplus;
```

Reply:
I swear I tried this, and the extra assignment canceled out the benefit, so I just left it. Now it seems to save 3 bytes, though. Well, it's clearer anyway.